### PR TITLE
Use default skip pattern for MP metrics, openapi, health

### LINKED ISSUE #89: Use default value for skip-patter
Skip pattern #16 could have a default value to exclude metrics, health and other endpoints.

### PR DESCRIPTION
**Pull request by pavolloffay**
Resolves #89 

Signed-off-by: Pavol Loffay <ploffay@redhat.com>

### CODE REVIEW

**Comment by pavolloffay** [comment]
To test this I have added servlets mapped to `/health`, `/openapi`... I am not sure whether this is the right way to go. Maybe do not include any servlets and just add dependency on APIs and expect vendor adds implementation when running the tests. But doing so means vendor has to provide an implementation for MP health, openapi and metrics.

File: spec/src/main/asciidoc/microprofile-opentracing.asciidoc [hunk 1]
```diff
@@ -192,6 +192,8 @@ Configuration of the skip pattern leverages MicroProfile Config specification.
 The skip pattern is specified as a string with key `mp.opentracing.server.skip-pattern` which has to be
 compliant with `java.util.regex.Pattern`. An example skip pattern might be `mp.opentracing.server.skip-pattern=/health|/metrics.*`
 
+The default value of the skip pattern excludes tracing for endpoints defined by MicroProfile Health, Config and OpenAPI specification.
```
Comment (kenfinnigan) [inline spec/src/main/asciidoc/microprofile-opentracing.asciidoc]: Should this be "Metrics" and not "Config"?

Comment (pavolloffay) [reply spec/src/main/asciidoc/microprofile-opentracing.asciidoc]: lol I don't know what I wanted to exclude in config

**Comment by kenfinnigan** [comment]
I think it's fine to add a servlet to test that the skip pattern works

**Comment by pavolloffay** [comment]
@fmhwong could you please also have a look?

File: tck/src/main/java/org/eclipse/microprofile/opentracing/tck/OpenTracingBaseTests.java [hunk 2]
```diff
@@ -80,6 +81,7 @@ public static WebArchive createDeployment() {
 
         WebArchive war = ShrinkWrap.create(WebArchive.class, "opentracing.war")
             .addPackages(true, OpenTracingClientBaseTests.class.getPackage())
+            .addClass(HealthServlet.class)
```
[the remark this answers is not in the document]
Comment (pavolloffay) [reply tck/src/main/java/org/eclipse/microprofile/opentracing/tck/OpenTracingBaseTests.java]: I will remove this. There are  added in `addPackages` and then auto-discovered. This was just my debug attempt :)

**Comment by fmhwong** [comment]
LGTM

**Pull request merged**